stock_picking_batch_creation/models/stock_picking_batch.py

```diff
@@ -24,10 +24,3 @@ class StockPickingBatch(models.Model):
         string="Number of picking lines",
         help="Indicates the picking lines ready for preparation.",
     )
-
```

Comment:
@sbejaoui This change breaks tests... When you write the user on the batch, it's not propagated to the pickings...